src/codeu/chat/client/ClientUser.java

```diff
@@ -49,10 +49,11 @@ static public boolean isValidName(String userName) {
     boolean clean = true;
     if (userName.length() == 0) {
       clean = false;
+    //if alphanumeric
```

Comment:
Omit comments that don't help explain the code. And when you do comment, use proper capitalization and punctuation, so
`// If alphanumeric.`